tests/tests-core/src/main/java/com/gentics/mesh/core/admin/ConsistencyCheckTest.java

```diff
@@ -57,7 +58,7 @@ public void testConsistencyRepair() {
 		assertEquals(ConsistencyRating.CONSISTENT, response.getResult());
 
 		tx(() -> {
-			content().removeElement();
+			((Node) content()).removeElement();
```

Comment:
Not sure if this test will apply to the enterprise implementation as well. If so, I have to find a way to remove this casting

Reply:
No need to. This is a pure OrientDB context, so the test has been moved accordingly.